### PR TITLE
Display 'No old trains' when old tab is selected

### PR DESCRIPTION
**Pull request by mathcolo**
My first attempt at a frontend thing in ages!
image

This change causes a visual artifact when switching from old to new, but I don't know React well enough to fix it. I think it's because the network request is slower than the tab switch, and it's re-rendering too early now?

### CODE REVIEW

File: src/components/Line.tsx [hunk 1]
```diff
@@ -8,12 +8,19 @@ import { renderTextTrainlabel } from '../labels';
...
+const AGE_WORD_MAP = new Map<VehiclesAge, string>([
+    ['new_vehicles', ' new '],
+    ['old_vehicles', ' old '],
+    ['vehicles', ' '],
+]);
+
 interface LineProps {
     api: MBTAApi;
     line: TLine;
+    age: string;
```
Comment (idreyn) [inline src/components/Line.tsx:23]: I think this can be of type `VehicleAge`, right?

**Review by idreyn — APPROVED** [review]
LGTM!

**Comment by lawrencet1** [comment]
Can we please wrap this up? Also, given that old trains are no longer running on the OL at all, can the OL old tab have it say something like "The MBTA has stopped running old trains on the Orange Line." please?

File: src/components/Line.tsx (hunk 1 above)
Comment (devinmatte) [reply src/components/Line.tsx:23]:
```suggestion
    age: VehiclesAge;
```

**Review by devinmatte — APPROVED** [review]
Looks good to go by me after this one Typescript fix

**Review by PatrickCleary — APPROVED** [review]

**Pull request merged**